docs/guide/comparisons.md

```diff
@@ -2,9 +2,7 @@
 
 ## WMR
 
-[WMR](https://github.com/preactjs/wmr) by the Preact team provides a similar feature set, and Vite 2.0's support for Rollup's plugin interface is inspired by it.
-
-WMR is mainly designed for [Preact](https://preactjs.com/) projects, and offers more integrated features such as pre-rendering. In terms of scope, it's closer to a Preact meta framework, with the same emphasis on compact size as Preact itself. If you are using Preact, WMR is likely going to offer a more fine-tuned experience.
+[WMR](https://github.com/preactjs/wmr) is unfortunately no longer maintained and the Preact team recommends to use Vite with [@preactjs/preset-vite](https://github.com/preactjs/preset-vite).
```

Comment:
@rschristian would you help us check what is the best wording here? I would like to keep the shoutout about Rollup's plugin API being inspired by WMR

```suggestion
[WMR](https://github.com/preactjs/wmr) by the Preact team provided a similar feature set. Vite's universal Rollup's plugin API for dev and build was inspired by it.

WMR is no longer maintained. The Preact team now recommends Vite with [@preactjs/preset-vite](https://github.com/preactjs/preset-vite).
```

Reply:
Looks good to me, though "Rollup's" should probably be "Rollup".

Certainly happy if you wanted to change that to "...Preact team looked to provide a similar feature set.", as I'm not sure we ever hit much parity.

Reply:
Thanks!